sagemaker-pyspark-sdk/src/sagemaker_pyspark/wrapper.py

```diff
@@ -157,3 +159,27 @@ def _to_java(self):
 
     def _from_java(self):
         raise NotImplementedError()
+
+
+# wrap scala.collection.immutable.List
+class ScalaList(SageMakerJavaWrapper):
+
+    def __init__(self, p_list):
+        self.p_list = p_list
+
+    def _to_java(self):
+        # Since py4j cannot deal with scala list directly
+        # we convert to scala listmap as an intermediate step
+        s_list = self._new_java_obj("scala.collection.immutable.ListMap")
```

Comment:
can you follow the pattern of adding a _wrapped_class class variable to ScalaList?  when we do _from_java() conversion SageMakerJavaWrapper relies on each subclass declaring its wrapped class to identify which python class to create.  I know _from_java() is not implemented now, but in the future it may come to bite us.

Reply:
_wrapped_class added. The reason I did not add was I did not use it for _new_java_object in this class.

Nice point that we may need this later for _from_java()